### ISSUE TITLE
[Talk Expert ✔️+(1)-803-993-9177] how To Cancel Bumble ™ subscription

### ISSUE THREAD

**Issue by travelsreadyfor** [typescript-eslint/typescript-eslint#10409]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3A%22package%3A+eslint-plugin%22) and found none that matched my issue.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Playground Link

https://github.com/typescript-eslint/typescript-eslint/issues/10402

### Repro Code

```TypeScript
To cancel your Bumble subscription:

    Open the Bumble App: Log into your account.
    Go to Settings: Tap on the profile icon in the bottom-left corner.
    Subscription Management: Scroll down and tap "Manage Payment" or "Subscription."
    Cancel Subscription: Choose the option to cancel. Confirm the cancellation when prompted.
    Check Confirmation: You should receive an email confirming the cancellation.

If you subscribed through iOS or Android, manage your subscription through the App Store or Google Play Store settings.
```

### ESLint Config

```javascript
module.exports = {
  parser: "@typescript-eslint/parser",
  rules: {
    "@typescript-eslint/<rule-name>": ["error", ...<options>],
  },
};
```

### tsconfig

```jsonc
{
  "compilerOptions": {
    // ...
  }
}
```

### Expected Result

[Talk Expert ✔️+(1)-803-993-9177] how To Cancel Bumble ™ subscription

### Actual Result

[Talk Expert ✔️+(1)-803-993-9177] how To Cancel Bumble ™ subscription
